tools/linter_lib/custom_check.py

```diff
@@ -376,12 +376,6 @@
             },
             "description": "Please use access_stream_by_*() to fetch Stream objects",
         },
-        {
-            "pattern": "datetime[.](now|utcnow)",
-            "include_only": {"zerver/", "analytics/"},
-            "description": "Don't use datetime in backend code.\n"
-            "See https://zulip.readthedocs.io/en/latest/contributing/code-style.html#naive-datetime-objects",
-        },
```

Comment:
So removing this linter rule causes us to no longer point to https://zulip.readthedocs.io/en/latest/contributing/code-style.html#naive-datetime-objects; thoughts? I worry a bit that a programmer not familiar with our codebase won't know how to fix the Ruff linter errors correctly, without this rule.

Reply:
Ruff’s messages make it sufficiently clear what to do.

``a.py:2:1: DTZ005 The use of `datetime.datetime.now()` without `tz` argument is not allowed``
``a.py:3:1: DTZ003 The use of `datetime.datetime.utcnow()` is not allowed. Use `datetime.datetime.now(tz=)` instead.``

Reply:
OK, WFM. We can always adjust if we see nonsense.